charts/kube-prometheus-stack/charts/crds/values.yaml

```diff
@@ -0,0 +1,7 @@
+enabled: true
+## The CRD upgrade job mitigates the limitation of helm not being able to upgrade CRDs.
```

Comment:
I would like to avoid the documentation twice. Can we say something like checkout kube-prometheus-stack values.yaml for more infomation?

Reply:
Done